crates/dslab-mp-python/python-tests/node.py

```diff
@@ -0,0 +1,45 @@
+from dslabmp import Context, Message, Process, StateMember
+from typing import List
+
+class Node(Process):
```

Comment:
Избегаем путаницы с node

```suggestion
class TestProcess(Process):
```
и файл надо переименовать в `process.py`.

Reply:
done